Documentation/advanced-configuration.md

```diff
@@ -433,3 +434,34 @@ other OSDs holding replica data are unavailable:
 ```bash
 ceph osd primary-affinity osd.0 0
 ```
+
+## Phantom OSD Removal
+
+If you have OSDs in which are not showing any disks, you can remove those "Phantom OSDs" by following the instructions below.
+To check for "Phantom OSDs", you can run:
```

Comment:
are there any other side effects of the phantom osd? Or you just notice them when you look at the osds with the ceph tools?

Reply:
@travisn The side effects seem to be nothing but also that the cluster may report an unhealthy status because of it.